spec/controllers/supervisor/dashboard_controller_spec.rb

```diff
@@ -1,7 +1,58 @@
 require 'spec_helper'
 
-describe 'routes for dashboard', type: :routing do
-  it 'routes /dashboard to the dashboard controller' do
-    expect(get('/supervisor/dashboard')).to route_to('supervisor/dashboard#index')
+describe Supervisor::DashboardController do
+  render_views
+  describe 'routes for dashboard', type: :routing do
+    it 'routes /dashboard to the dashboard controller' do
+      expect(get('/supervisor/dashboard')).to route_to('supervisor/dashboard#index')
+    end
+    it 'routes /supervisor to the dashboard controller' do
+      expect(get('/supervisor')).to route_to('supervisor/dashboard#index')
+    end
   end
-end
\ No newline at end of file
+end
+#
+#
+#
+#
+#     context "with non-supervisor logged in"
+#       it "should redirect non-supervisors"
+#       it "should redirect supervisors on a different team"
+#       it "should redirect supervisors on past season's teams"
+# end
+#
+  # describe 'access for supervisors only', type: :controller do
+  #   context 'with supervisor logged in' do
+  #     let(:current_user) { build_stubbed(:user) }
+  #
+  #     before do
+  #       allow(current_user).to receive(:supervisor?).and_return(true)
+  #       allow(controller).to receive(:current_user).and_return(current_user)
+  #     end
+  #     it 'renders the dashboard (index template)' do
+  #       get :index
+  #       expect(response).to be_success
+  #       expect(response).to render_template :index
+  #     end
+  #   end
+  # end
+#end
+
+
+#   describe 'access for supervisors only' do
+#     context 'with supervisor logged in' do
+#       let(:current_user) { build_stubbed(:user) }
+#
+#       before do
+#         allow(current_user).to receive(:supervisor?).and_return(true)
+#         allow(controller).to receive(:current_user).and_return(current_user)
+#       end
+#
+#       it 'renders the dashboard (index template)' do
+#         #get '/supervisor/dashboard#index'
+#         #expect(response).to be_success
+#         #expect(response).to render_template 'index'
+#       end
+#     end
+#   end
+# end
```

Comment:
Commenting out code serves little purpose and only adds Git diff noise. If this code isn't needed, get rid of it. If it's meant for later then it should be committed once it's needed/working/etc.